library/designsystem/src/main/java/ir/composenews/designsystem/component/FavoriteIcon.kt

```diff
@@ -40,7 +39,7 @@ fun FavoriteIcon(
         updateTransition(targetState = animationState, label = "favorite_animation_state")
 
     val colorTintFavorite by favoriteTransition.animateColor(label = "favorite_color") { state ->
-        if (state) MaterialTheme.colors.error else Color.LightGray
+        if (state) Color.Red else Color.LightGray
```

Comment:
Did you test it in Dark mode? I set `MaterialTheme.colors.error` to let OS decide which color set.

Reply:
Yes, I did. First, I put " **Material Theme.colors.error** " on this mode so that the operating system decides, but in dynamic color mode, some colors were not interesting. Anyway, this " **MaterialTheme.colors.error** " should be set and that's the correct one.